packages/core/src/network-resolver.test.ts

```diff
@@ -0,0 +1,80 @@
+import {createWallet, IWallet} from './wallet';
+import {dockDocumentNetworkResolver} from './network-resolver';
+
+describe('Wallet', () => {
+  let wallet: IWallet;
+
+  beforeEach(async () => {
+    wallet = await createWallet({
+      databasePath: ':memory:',
+      documentNetworkResolver: dockDocumentNetworkResolver,
+    });
+  });
+
+  it('expect to add document to mainnet', async () => {
+    const mockDocument = {
+      id: 'test',
+      type: 'VerifiableCredential',
+    };
+
+    await wallet.addDocument(mockDocument);
+
+    const [result] = await wallet.getDocumentsByType(mockDocument.type);
+    expect(result.networkId).toEqual('mainnet');
+    expect(result.id).toEqual(mockDocument.id);
+  });
+
+  it('expect to add document to testnet', async () => {
+    const mockDocument = {
+      id: 'test',
+      type: 'VerifiableCredential',
+    };
+
+    wallet.setNetworkId('testnet');
+
+    await wallet.addDocument(mockDocument);
```

Comment:
is it worth asserting that the document _wasn't_ added to the other network?

Reply:
sure, added it here https://github.com/docknetwork/react-native-sdk/pull/115/commits/eda903201840da84eefb332862055a3d49fcd897